### ISSUE TITLE
Curation module: Generate tasks based on the before Geo-Object in a transition rather than the after Geo-Objects

### ISSUE THREAD

**Issue by SteeveEbener** [terraframe/geoprism-registry#696]
**List the CGR information**
- CGR version: 0.17.0
- CGR instance (e.g. a URL): Staging

**Describe the bug (clear and concise)**
In the case of a split, the curation module contains a task for each of the resulting geo-object creating confusion regarding which action needs to be taken.

**To Reproduce**
As the RA/RM: 
1. Go to Historical events module
2. Look for the tasks linked to a split
3. See error: there is one task for each of the resulting split geo-object (see screenshot below)

**Expected behavior (clear and concise)**
There is only one task captured in the curation module in case of a split. The message in this case could read for example read like this using the example below and the content of ticket #694:

"Please check if some of the [County (MOHA)], children of Geo-Object [Andustar] of type [Province [MOHA)] before the event, should be reassigned to one of the [Province (MOHA)] resulting from the split in hierarchy [Health Facility Management Hierarchy VR] starting from [4/20/07]"

**Screenshots**
![image](https://user-images.githubusercontent.com/12602163/141603144-f8e17c45-bfe3-4c7f-81d8-f040f503e41f.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: latest

**Additional context (if any)**

**Comment by jsmethie** [comment]
I do not see duplicate tasks.  In your screen shot the target geo objects are different.  The geo object has split into both Orrostar and Andustar. The children for each need to be reassigned.

**Comment by SteeveEbener** [comment]
The message is not meant to be based on the after (target) geo-object but on the before one (Nate's comment in the google doc was unfortunately incorrect) and this for the simple reason that, at the time of the event, the children are attached to the before geo-object => they need to be reassigned to the after ones => need only one message.

Good point regarding the need for the message to cover case with more than 2 resulting geo-object after the split => revised message:
 
"The Geo-Object [Andustar] of type [Province (MOHA)] has split into the Geo-Objects [Orrostar, Andustar] of type [Province (MOHA)]. Please check if some of the [Health Facility VR], children of Geo-Object [Andustar] of type [Province (MOHA)] before the event, should be reassigned to one of the parent(s) resulting from the split in hierarchy [Health Facility Management Hierarchy VR] starting from [4/20/07]"

**Comment by jsmethie** [comment]
The tasks must be made in regards to the target geo object because the person who has the ability to reassign the children to the new type is the person with the mandate over the target geo object type.

**Comment by SteeveEbener** [comment]
Nope, the person having the curation mandate over the children of the target geo-object type actually + this person will find the children in question attached the before "geo-object", not the after one => what I am saying is correct, the message has to be related to the before geo-object, not the after one

**Comment by SteeveEbener** [comment]
Can the "invalid" tag please be removed from this ticket? This ticket is valid. Thanks

**Comment by justinlewis** [comment]
I updated the title of this ticket to better reflect what is being asked.

**Comment by justinlewis** [comment]
@steeveebener and I talked outside of this ticket. He said he would like to see an approach that follows this logic:

> 1. identify the hierarchy(ies) in which the before geo-object participates; 
> 2. identify the children geo-object types in each of these hierarchies; 
> 3. Create a task for each of these children; 
> 4. Display the task only to the concerned RA/RMs (those having the curation mandate over these children geo-object types)

**Comment by SteeveEbener** [comment]
@justinlewis : has the approach described in your last message here above finally been implemented?

**Comment by justinlewis** [comment]
Closing legacy issues created under a prior funded development effort. These items were associated with a separate roadmap and stakeholder group targeting the v 1.x release. Archiving these tickets enables a v 2 reset of the backlog. All of these tickets can be found by searching for the geoprism-v1-archive tag on tickets. If any issue remains relevant to current priorities, please reopen or ideally create a new issue referencing this thread.
